stdlib/influxdata/influxdb/alerts/alerts.flux

```diff
@@ -0,0 +1,9 @@
+package alerts
+
+builtin check
+
+write = (tables=<-) => tables |> to(bucket: "system")
```

Comment:
I think this will be implemented in Go now.

Reply:
It will be `experimental`, right?